sahi/slicing.py

```diff
@@ -258,7 +258,6 @@ def slice_image(
     sliced images.
 
     Args:
-        auto_slice_resolution:
```

Comment:
is this mistakenly removed ?

Reply:
no i deleted it since it is a redundant line

Reply:
fixed it